packages/js/README.md

```diff
@@ -239,17 +187,21 @@ Screenshot:
 
 #### Video call
 
-```
-cd examples/react-video
-npm run setup
-npm install
-npm start
+```sh
+$ cd examples/react-video
+$ npm run setup
+$ npm install
+$ npm start
```

Comment:
```suggestion
cd examples/react-video
npm run setup
npm install
npm start
```

Reply:
If you intended to keep the `$` to make it clear that those are separate commands, you can add `;` at the end of each line